e2e/README.md

```
@@ -0,0 +1,43 @@
+```
+Pull image from github, then run
+
+```
+npm run e2e:debug
+```
+
+Open the Cypress client for debugging.
+Cypress will capture screenshots of your case pages.
+Review the screenshots in e2e/downloads folder, store them in the e2e/fixtures/originImage directory if there are no issues, then rerun the test cases. If the test cases pass, the debugging is complete.
+
+### 4. Run the complete e2e tests:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)